src/relayserver/HttpServer.ts

```diff
@@ -30,16 +31,16 @@ export class HttpServer {
 
   start (): void {
     if (this.serverInstance === undefined) {
-      this.serverInstance = this.app.listen(this.port, () => {
+      this.serverInstance = this.app.listen(this.port, async () => {
         console.log('Listening on port', this.port)
-        this.startBackend()
+        await this.startBackend()
```

Comment:
also, here we are using `await` and therefore returning a promise, but according to the `listen` signature this argument should return `Void`

Reply:
I don't see the return statement, where is it?

Reply:
You are right here btw

Reply:
Basically listen has all these signatures:
`
listen(port: number, hostname: string, backlog: number, callback?: () => void): http.Server;
    listen(port: number, hostname: string, callback?: () => void): http.Server;
    listen(port: number, callback?: () => void): http.Server;
    listen(callback?: () => void): http.Server;
    listen(path: string, callback?: () => void): http.Server;
    listen(handle: any, listeningListener?: () => void): http.Server;
`
And we don't have any async implementation for those, so i will remove the async and just call startBackend like it was before